spec.bs

```diff
@@ -1024,73 +1116,185 @@ Issue: Once <a href="https://github.com/wicg/turtledove/issues/615">
     {{InterestGroupScriptRunnerGlobalScope}}s until the script's initial
     execution is complete.
 
-<div algorithm>
-The <dfn method for="PrivateAggregation">contributeToHistogramOnEvent(DOMString
-event, PAExtendedHistogramContribution contribution)</dfn> method steps are:
-</div>
-
-Issue: Fill in validation. Will need to check offsets are valid and in the right
-    range for the type etc. Also that base values are one of a set.
+Protected Audience API-specific structures {#protected-audience-api-specific-structures}
+----------------------------------------------------------------------------------------
 
-Issue: Need to document limits on offset, etc.
+<h4 id="extending-auction-config">Extending auction config</h4>
 
-Issue: Fill in the rest. (Need to put the contribution in some sort of queue and
-    process the queue at some point. Need to decide where the queue should live
-    given it has to outlive the auction.)
+Extend the <a spec="turtledove">auction config</a> [=struct=] to add a new
+field:
+: <dfn for="auction config" spec="turtledove">per-bid or seller on event
```

Comment:
is it better to define these as fields of InterestGroupScriptRunnerGlobalScope, similar to https://github.com/WICG/turtledove/blob/main/spec.bs#L1914?
This seems to be the only things needed from auctionConfig, so we didn't add auctionConfig to GlobalScope, and you cannot retrive auction config from GlobalScope, unless this spec or protected audience spec adds it to GlobalScope.

Reply:
I think these need to be bound to an auction as a whole rather than just a global scope as the spec doesn't keep track of which global scopes have been used in the past, so when we 'fill in' these contributions later, the global scopes might not be around any more.

But, yes, I think you're right that we should be exposing these values on the global scope itself so we don't need to use this handwavy prose. I've added an issue and plan to follow up in a separate PR shortly if that's ok with you.